packages/mgt-spfx/README.md

```diff
@@ -2,94 +2,4 @@
-
-If you need to use a different version of MGT other than the one supplied by the centrally deployed version of `mgt-spfx` then please refer to the documentation for [disambiguation](https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/main/packages/mgt-components#disambiguation) and [`mgt-spfx-utils`](https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/main/packages/mgt-spfx-utils).
-
-## Usage
-
-When building SharePoint Framework web parts and extensions, reference the Microsoft Graph Toolkit `Provider` and `SharePointProvider` from the `@microsoft/mgt-spfx` package. This will ensure, that your solution will use MGT components that are already registered on the page, rather than instantiating its own. The instantiation process is the same for all web parts no matter which JavaScript framework they use:
-
-```ts
-import { Providers, SharePointProvider } from '@microsoft/mgt-spfx';
-
-// [...] trimmed for brevity
-
-export default class MgtWebPart extends BaseClientSideWebPart<IMgtWebPartProps> {
-  protected async onInit() {
-    if (!Providers.globalProvider) {
-      Providers.globalProvider = new SharePointProvider(this.context);
-    }
-  }
-
-  // [...] trimmed for brevity
-}
-```
-
-When building web parts using framework other than React, you can load components directly in your web part:
-
-```ts
-export default class MgtNoFrameworkWebPart extends BaseClientSideWebPart<IMgtNoFrameworkWebPartProps> {
-  protected async onInit() {
-    if (!Providers.globalProvider) {
-      Providers.globalProvider = new SharePointProvider(this.context);
-    }
-  }
-
-  public render(): void {
-    this.domElement.innerHTML = `
-      <div class="${styles.mgtNoFramework}">
-        <div class="${styles.container}">
-          <div class="${styles.row}">
-            <div class="${styles.column}">
-              <span class="${styles.title}">No framework webpart</span>
-              <mgt-person person-query="me" show-name show-email></mgt-person>
-            </div>
-          </div>
-        </div>
-      </div>`;
-  }
-
-  // [...] trimmed for brevity
-}
-```
-
-If you build web part using React, load only components from the `/dist/es6/spfx` path in the `@microsoft/mgt-react` package:
-
-```tsx
-import { Person } from '@microsoft/mgt-react/dist/es6/spfx';
-
-// [...] trimmed for brevity
-
-export default class MgtReact extends React.Component<IMgtReactProps, {}> {
-  public render(): React.ReactElement<IMgtReactProps> {
-    return (
-      <div className={ styles.mgtReact }>
-        <Person personQuery="me" />
-      </div>
-    );
-  }
-}
-```
+⚠️⚠️⚠️ This package is deprecated. It is no longer available from v4.0.0 onwards. All new applications should use the [disambiguation approach](https://learn.microsoft.com/graph/toolkit/customize-components/disambiguation) instead ⚠️⚠️⚠️
```

Comment:
Should we offer a link to the repo files in the v3 branch?

Reply:
Oh, that is a good idea.

Reply:
@sebastienlevert I've added a link to the tag for the last release.